h2o-core/src/main/java/water/server/ServletUtils.java

```diff
@@ -122,6 +122,16 @@ public static void sendResponseError(HttpServletResponse response, int sc, Strin
     response.sendError(sc, msg);
   }
 
+  public static InputStream extractInputStream(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    final InputStream is;
```

Comment:
I think that local variable definition is redundant, result could be returned directly from if-else branches, but leaving that up to you

Reply:
thank you, in this case, I did it on purpose for easy of debugging